tests/components/trafikverket_camera/conftest.py

```diff
@@ -36,6 +36,10 @@ async def load_integration_from_entry(
         title="Test location",
     )
 
+    # Wait to ensure the recorder mock is finished
+    await hass.async_block_till_done()
+    await hass.async_block_till_done()
```

Comment:
Question out of curiosity, what does async block till done do? I would expect it waits till it's done but why do we call it twice? Is there an easy explanation?

Reply:
It waits until all tasks are done. If a task spawns other tasks we need to call it again (and again) to ensure all tasks are done.
The real reason for the failing test is fixed here: https://github.com/home-assistant/core/pull/99080